exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,25 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
```

Comment:
what do you expect from the function return? Did you try to see if your function works?